src/Codeception/Lib/Console/MessageFactory.php

```diff
@@ -22,38 +22,26 @@ class MessageFactory
      */
     protected $colorizer;
 
-    /**
-     * MessageFactory constructor.
-     * @param Output $output
-     */
     public function __construct(Output $output)
     {
         $this->output = $output;
         $this->diffFactory = new DiffFactory();
         $this->colorizer = new Colorizer();
     }
 
-    /**
-     * @param string $text
-     * @return Message
-     */
-    public function message($text = '')
+    public function message(string $text = ''): Message
     {
         return new Message($text, $this->output);
     }
 
-    /**
-     * @param ComparisonFailure $failure
-     * @return string
-     */
-    public function prepareComparisonFailureMessage(ComparisonFailure $failure)
+    public function prepareComparisonFailureMessage(ComparisonFailure $failure): string
     {
         $diff = $this->diffFactory->createDiff($failure);
         if (!$diff) {
```

Comment:
Changing this condition to `$diff !== ''` would allow to simplify the signature of createDiff method.